delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
```

Comment:
The code for deleting a book is vulnerable to SQL injection. Use prepared statements to mitigate this risk.

```php
if (isset($_GET['id'])) {
    $id = intval($_GET['id']); // Ensure ID is an integer
    $stmt = $connection->prepare("DELETE FROM books WHERE id = ?");
    $stmt->bind_param("i", $id);
    if ($stmt->execute()) {
        echo "Book deleted successfully!";
        header("Location: list_books.php");
    } else {
        echo "Error: " . $stmt->error;
    }
    $stmt->close();
}
```